src/deploy/NVA_build/Base.Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM noobaa/builder
```

Comment:
if noobaa/builder does not exist in the remote registry there is no reason to name it `noobaa/builder`. if it will only exist locally just name it `noobaa-builder`